icons/plane-tickets.json

```
@@ -0,0 +1,15 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "jguddas"
+  ],
+  "tags": [
+    "plane",
+    "trip",
+    "airplane"
```

Some extra tags to consider:
```suggestion
    "airplane"
    "flight",
    "travel",
    "fly",
    "takeoff",
    "vacation",
    "passenger",
    "pass",
    "check-in",
    "airport"
```